### PR TITLE
Add information about the Coalesce function

### PR DESCRIPTION
**Pull request by sarahrudy**
Add info about coalesce function for Destination Actions

### Proposed changes

Add info about coalesce function for Destination Actions

### Merge timing
- ASAP once approved

### CODE REVIEW

File: src/connections/destinations/actions.md [hunk 1]
```diff
@@ -117,6 +117,13 @@ If necessary, click **New Mapping** to create a new, blank action.
 > info ""
 > The required fields for a destination mapping appear automatically. Click the + sign to see optional fields.
 
+### Coalesce Function
```
Comment (stayseesong) [inline src/connections/destinations/actions.md:120]:
```suggestion
### Coalesce function
```

File: src/connections/destinations/actions.md [hunk 2]
```diff
@@ -117,6 +117,13 @@ If necessary, click **New Mapping** to create a new, blank action.
...
+
+The coalesce function will take a primary value and use that one if it is available. In the case it is not, it will use the fallback value.
```
Comment (stayseesong) [inline src/connections/destinations/actions.md:122]:
```suggestion
The coalesce function takes a primary value and uses it if it is available. If the value isn't available, the function uses the fallback value instead. 
```

File: src/connections/destinations/actions.md [hunk 3]
```diff
@@ -117,6 +117,13 @@ If necessary, click **New Mapping** to create a new, blank action.
 > info ""
 > The required fields for a destination mapping appear automatically. Click the + sign to see optional fields.
 
+### Coalesce Function
+
+The coalesce function will take a primary value and use that one if it is available. In the case it is not, it will use the fallback value. 
+
+![coalesce](https://user-images.githubusercontent.com/78389005/213260648-a9eda0c8-fd88-4a0b-b4c6-682380ce2328.png)
```
Comment (stayseesong) [inline src/connections/destinations/actions.md:124]:
```suggestion
```

Comment (stayseesong) [reply src/connections/destinations/actions.md:124]: I don't think we need to include a screenshot to this as it doesn't add any value

File: src/connections/destinations/actions.md (hunk 1 above)
Comment (stayseesong) [reply src/connections/destinations/actions.md:120]: @sarahrudy how do you access the coalesce function? I don't see it in the UI

**Review by stayseesong — CHANGES_REQUESTED** [review]

**Review by markzegarelli — APPROVED** [review]

**Review by stayseesong — APPROVED** [review]

**Pull request merged**